#! /bin/sh /usr/share/dpatch/dpatch-run
## fix_Beagle.Search.exe.config.dpatch by Christopher James Halse Rogers <raof@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add dllmaps for Beagle.Search.exe, and fix SONAMEs.

@DPATCH@
diff -urNad beagle-0.3.9~/search/Beagle.Search.exe.config beagle-0.3.9/search/Beagle.Search.exe.config
--- beagle-0.3.9~/search/Beagle.Search.exe.config	2008-07-17 09:59:14.000000000 +0100
+++ beagle-0.3.9/search/Beagle.Search.exe.config	2009-04-19 13:14:42.000000000 +0100
@@ -1,4 +1,5 @@
 <configuration>
-	<dllmap dll="libgnome-desktop-2" target="libgnome-desktop-2.so.2" />
+	<dllmap dll="libgnome-desktop-2" target="libgnome-desktop-2.so.17" />
+	<dllmap dll="libbeagleuiglue.so" target="/usr/lib/beagle/libbeagleuiglue.so"/>
 </configuration>
 
